core/server/errors/database-version.js

```diff
@@ -0,0 +1,11 @@
+function DatabaseVersion(message) {
+    this.message = message;
+    this.stack = new Error().stack;
+    this.statusCode = 400;
+    this.errorType = this.name;
+}
```

Comment:
Again this seems like overkill having a specific type of error, and also seems to me to be a server issue (500) not a user issue (400)?